engine/engine/content/builtins/manifests/web/engine_template.html

```diff
@@ -69,6 +69,11 @@
 </head>
 
 <body>
+	<div id="running-from-file-warning" style="display: none; margin: 3em;">
```

Comment:
should we exclude it for release?

Reply:
Possibly. Or have it as an option in game.project?

Reply:
hmmm maybe, but in this case user will never know we have something like this in index.html

Reply:
One concern I have with removing this warning in a release build is that a beginner might export a release build and try to double-click on index.html and end up in the same situation as we're trying to resolve.

Reply:
true